course3/meeting2/hello.cc

```diff
@@ -1,13 +1,56 @@
 #include <iostream>
-#include <string>
 
+class SimpleString
+{
+  public:
+    SimpleString()
+    {
+    }
+
+    SimpleString(const char* ascii)
+    {
+      int lastCharacterIndex = BUFFER_SIZE - 1;
+
+      int i = 0;
+      while(i < lastCharacterIndex
+            && ascii[i] != 0)
+      {
+        mBuffer[i] = ascii[i];
+	++i;
+      }
+      mBuffer[i] = 0;
+    }
```

Comment:
Is the type copyable? Is it assignable? Have you tried compiling the following code:
```
SimpleString a("hello");
SimpleString b = a;
```
or
```
SimpleString a("hello");
SimpleString b;
b = a;
```